std/meta.d

```diff
@@ -61,6 +61,67 @@ unittest
 }
 
 /**
+ * Not everything can be directly aliased. An alias cannot be declared
```

Comment:
I agree, but I can't come up with a good way of putting it. `Alias` seems most succinctly defined by what you can't do without it.